app/entry.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('./css/base.scss');
+
+const angular = require('angular');
+angular.module('app', [])
+.component('app', {
+  template: require('./app.html'),
+  controller: ['$log', function($log){
```

Comment:
You don't need Lines 9-15 if you're not doing any extra functionality in this controller.